tests/EventTest.php

```diff
@@ -119,32 +119,27 @@ public function testToArray(): void
         $this->assertEquals($expected, $event->toArray());
     }
 
-    public function testToArrayWithMessage(): void
+    /**
+     * @dataProvider toArrayWithMessageDataProvider
+     */
+    public function testToArrayWithMessage(array $message, $expectedValue): void
```

Comment:
Please rename this variable to something that better reflect what it stores (e.g. `$setMessageMethodArguments`)